game/script-jokes.rpy

```diff
@@ -735,3 +755,56 @@ label joke_cheesy_pizza:
     extend 1nsqpo " {i}that cheesy{/i}."
     
     return
+
+label joke_veggie_mood:
+    n 1nslsssbl "...I've found a joke reminding me of Monika."
+    n 1ccsajsbl "E-{w=0.2}ehm..."
+    n 1tnmaj "You've got {i}any{/i} idea when vegetarians experience mood swings?{w=0.75}"
+    n 1nllss "Well...{w=0.5}{nw}"
+    extend 1nslss " I do."
+    n 1nlraj "Obviosuly,{w=0.2} when they are out of their...{w}{nw}"
+    extend 1nwlbg " {i}gourd{/i}."
+    
+    return
+
+label joke_scarecrow_award:
+    n 1fsqfl "...Why did the scarecrow get an award?"
+    n 1ftlemesi "..."
+    n 1fcsbo "Because...{w=1}"
+    n 1fsqbo "Because it was outstanding in its field."
+    
+    return
+
+label joke_sundae_school:
+    n 1nslss "Heh...{w=0.5}"
+    n 1tnmbo "You know, I did some thinking about schools and stuff."
+    n 1nlraj "There were many schools I could've attended.{w=0.5}{nw}"
+    extend 1uwdaj " One even teached how to make ice cream!"
+    n 1tnmss "Oh?{w=0.3} You wonder which one would do that?{w=0.3}"
+    n 1nslss "It was the...{w=1}{nw}"
+    extend 1fsqcs " {i}sundae{/i}{w=0.75} school."
+    
+    return
+
+label joke_burned_tongue:
+    n 1fnmfl "Do you know these people always trying to gatekeep simply {i}everything{/i}?{w=0.3}{nw}"
+    extend 1fsran " It really grinds my gears!"
+    n 1fcsbo "But...{w=0.5}{nw}"
+    extend 1ncsaj " I guess it's fine."
+    n 1nllpu "I saw one of these hipsters in a café lately.{w=0.3}{nw}"
+    extend 1unmaj " They burned their tongue.{w=0.3}{nw}"
+    extend 1tnmss " Wondering why?{w=0.3}"
+    n 1flrss "Because they drank their coffee{w=1}"
+    extend 1fchgn " {i}before it was cool{/i}."
+
+    return
+
+label joke_pointless_pencil:
+    n 1fllfl "Man...{w=0.5}"
+    n 1fcsem "When I was working on my last poem,{w=0.2} my pencil just decided to break!{w=0.5}{nw}"
```

Comment:
```suggestion
    n 1fcsem "I was {i}trying{/i} to work on my poetry,{w=0.2} and my pencil just decided to break!{w=0.75}{nw}"
```

I think this would sound a bit smoother! It also makes more sense in the event that the player might not have seen a poem from Natsuki yet, as the wording makes it sound like she's talking about something she might have written for the player in the past!